lib/matplotlib/legend.py

```diff
@@ -384,8 +384,13 @@ def __init__(self, parent, handles, labels,
         # init with null renderer
         self._init_legend_box(handles, labels, markerfirst)
 
+        # If shadow is activated use framealpha if not
```

Comment:
Always glad to help.  :+1: 

I would put it into line
https://github.com/matplotlib/matplotlib/blob/5a241211a4000719eeee83954201786486bac54e/lib/matplotlib/axes/_axes.py#L427
to make clear that if shadow is activated and framealpha is  ```None```  it will set the value to be 1  instead of taking it from ``legend.framealpha`` :data:`rcParam<matplotlib.rcParams>`. 

Should I also add something to 
https://github.com/matplotlib/matplotlib/blob/5a241211a4000719eeee83954201786486bac54e/lib/matplotlib/axes/_axes.py#L422 ?